### ISSUE TITLE
V 0.16.0 Unstun does not remove 

### ISSUE THREAD

**Issue by Razortide** [SalieriC/SWADE-Immersive-Macros#65]
Tested with: 9.255, SWADE 1.0.8

Stun applies wounds as it should, but if you roll for unstun it marks the status(es) as removed, but they are still active:

![image](https://user-images.githubusercontent.com/35796457/161268762-c0dba9a0-8b5f-4c67-a631-47ce44748561.png)

**Comment by SalieriC** [comment]
You are right, but I don't get why. It's all there: https://github.com/SalieriC/SWADE-Immersive-Macros/blob/7f79a698584771fb05fc0699c7fe38c6a9293de3/swim/scripts/swim_modules/unstun.js#L74-L80
🤔

**Comment by SalieriC** [comment]
Apparently the script is faster than its shadow. A little delay did the trick. A hacky solution but at least it seems to work.
